src/ICeWall/doc/add_libcell.md

```diff
@@ -0,0 +1,60 @@
+## add_libcell
+### Synopsis
+```
+  % add_libcell \
+        [-name name] \
+        [-type cell_type] \
```

Comment:
for the "special" types: fill corner bondpad bump, it might make sense to have those be indicated as -bondpad -corner -fill -bump instead of by type since typos in "fill" vs "filler" doesn't generate an error or warning, but also doesnt allow the user to define the "fill" type name.

Reply:
So, -fill would be synonymous with -type fill etc., I can see how to add that.
Probably also makes sense to add a check at the start of initialize_padring to ensure that types for fill, and corner are defined, and that a type for bump is also defined for flipchip.